<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SportsDialog</class>
 <widget class="QDialog" name="SportsDialog">
  <property name="windowTitle">
   <string>Manage Sports</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit">
           <property name="maxLength">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="colorLabel">
           <property name="text">
            <string>Color: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="colorLineEdit">
           <property name="maxLength">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="choosePushButton">
           <property name="text">
            <string>C&amp;hoose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="sportsTreeWidget">
       <property name="columnCount">
        <number>3</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">3</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SportsDialog</receiver>
   <slot>slotAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SportsDialog</receiver>
   <slot>slotOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SportsDialog</receiver>
   <slot>slotCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SportsDialog</receiver>
   <slot>slotRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SportsDialog</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>choosePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SportsDialog</receiver>
   <slot>slotSetColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sportsTreeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>SportsDialog</receiver>
   <slot>slotSetNameText(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOk()</slot>
  <slot>slotAdd()</slot>
  <slot>slotRemove()</slot>
  <slot>slotSave()</slot>
  <slot>slotCancel()</slot>
  <slot>slotSetColor()</slot>
  <slot>slotSetNameText(QTreeWidgetItem*,int)</slot>
 </slots>
</ui>
